tools/test/examples/update.py

```diff
@@ -480,6 +482,8 @@ def check_update_status(examples, github, tag):
                                      formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument('-c', '--config_file', help="Path to the configuration file (default is 'examples.json')", default='examples.json')
     parser.add_argument('-T', '--github_token', help="GitHub token for secure access")
+
+    parser.add_argument('-S', '--use_ssh', help="Updates examples using ssh instead of https urls")
```

Comment:
what's the default value - behavior ?

Reply:
default is https as shown on line 350 @0xc0170